### ISSUE TITLE
Add tx from Foreign to Home

### ISSUE THREAD

**Issue by rstormsf** [poanetwork/bridge-e2e#5]
Currently the script only send a tx from Home to Foreing.
It doesn't send ERC20 back to Home

**Comment by fvictorio** [comment]
Yes, this is just a proof of concept. The next step would be to use a proper test framework (probably mocha or jest) and, after setting up everything, to run different tests (sending from home to foreign, from foreign to home, testing that txs are not sent if there aren't enough validations, etc.)
